### PR TITLE
Copy project files

### PR DESCRIPTION
**Pull request by nelsinofrancisco**

### CODE REVIEW

File: src/styles.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+.line-trought,
+.line-trought::placeholder {
+  text-decoration: line-through;
+}
+
+.text-description,
+.text-description::placeholder,
+.text-description:focus {
+  color: #6c757d;
```
Comment (VanessaAoki) [inline src/styles.css]: You could create a variable for this color #6c757d (you used it again in line 45);

File: src/styles.css [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  border: none;
+  outline: none;
+  width: 100%;
+}
+
+.arrow-icon {
+  right: 18px;
+  top: 6px;
+  width: 24px;
+  z-index: 2;
```
Comment (VanessaAoki) [inline src/styles.css]: You could create a variable to hold the index and its value, this is reused in line 44.

File: src/index.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      <h1 class="fs-3 text-secondary">All Tasks</h1>
+      <img src="#" class="icon sync-icon fas fa-sync-alt text-secondary" alt=""/>
+    </div>
+    <form class="m-auto px-2 d-flex position-relative">
+      <input id="add-task" class="w-100 p-2" type="text" placeholder="Add a task to the list">
+      <img src="#" class="icon arrow-icon position-absolute fas fa-2x fa-arrow-right" alt=""/>
+    </form>
+    <div id="todo-list-container" class="m-auto my-2 px-2">
+    </div>
+    <div>
```
Comment (VanessaAoki) [inline src/index.html:22]: I'm not sure if this div is really necessary, maybe you could keep only the button inside it.

File: src/modules/handle_tasks.js [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+      if (e.key === 'Enter' && e.target.value) {
+        const obj = {
+          description: e.target.value,
+          completed: false,
+          index: i,
+        };
+        this.input.value = '';
+        this.store.add(obj);
+        this.UI.generateHtmlTasks();
```
Comment (VanessaAoki) [inline src/modules/handle_tasks.js]: You could create a function to store these lines, you reuse the exact same ones in lines 35-37.

**Review by VanessaAoki — CHANGES_REQUESTED** [review]
Hi @nelsinofrancisco,

First of all, I would like to congratulate you on your amazing project!
I really like how your files are so well organized and you keep everything simple.

I do have some suggestions, I marked them as comments in your code, please consider going through them.

Keep up the great work!

File: src/styles.css (hunk 1 above)
Comment (nelsinofrancisco) [reply src/styles.css]: - [x] change implemented

File: src/styles.css (hunk 2 above)
Comment (nelsinofrancisco) [reply src/styles.css]: - [x] change implemented

File: src/index.html (hunk 3 above)
Comment (nelsinofrancisco) [reply src/index.html:22]: This div works as a placeholder for the tasks, so it's necessary!

File: src/modules/handle_tasks.js (hunk 4 above)
Comment (nelsinofrancisco) [reply src/modules/handle_tasks.js]: - [x] change implemented

**Review by VanessaAoki — APPROVED** [review]
@nelsinofrancisco,

Congratulations on implementing all the requested changes!
I'm approving your pull request! You may merge it now!

Keep up the great work!

**Pull request merged**